<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="HomePage">
       <widget class="QLabel" name="CurrentDate_Label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string>Current Date &amp; Time</string>
        </property>
       </widget>
       <widget class="QPushButton" name="playSoundBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>80</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Play Sound</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="soundComboBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Assembly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reveille and To the Colors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ACQ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Taps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First Call</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6-8 Drum Cadence, 4 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cadence (BCT), 8 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>National Anthem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Retreat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>To the Colors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bonk</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="playingLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="text">
         <string> Currently playing: None</string>
        </property>
       </widget>
       <widget class="QPushButton" name="stopSoundBtn">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>100</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop Sound</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="addSoundBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Sound</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="timeInput">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="scheduledSounds">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>256</width>
          <height>391</height>
         </rect>
        </property>
        <property name="resizeMode">
         <enum>QListView::ResizeMode::Adjust</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="removeSoundBtn">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>250</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove Sound</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="nextSound">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>251</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Playing next: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="currTemplate">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>251</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Today's Template: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QTextEdit" name="volumeInputHome">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="NextDayCheckHome">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Play Next Day</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="repeatBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Repeat</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="CalendarPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="templateDropdownCalendar"/>
        </item>
        <item>
         <widget class="QPushButton" name="setTemplateCalendar">
          <property name="text">
           <string>Set Template</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resetCalendarBtn">
          <property name="text">
           <string>Reset to Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="calendarWidget">
          <property name="gridVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Templates">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>471</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="text">
         <string>Templates</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::MarkdownText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
       <widget class="QListWidget" name="templateSoundSchedule">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>30</y>
          <width>256</width>
          <height>331</height>
         </rect>
        </property>
        <property name="resizeMode">
         <enum>QListView::ResizeMode::Adjust</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="soundComboBoxTemplate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Assembly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reveille and To the Colors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ACQ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Taps</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>First Call</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6-8 Drum Cadence, 4 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Retreat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cadence (BCT), 8 minutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>National Anthem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>To the Colors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bonk</string>
         </property>
        </item>
       </widget>
       <widget class="QTextEdit" name="timeInputTemplate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="removeSoundBtnTemplate">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>130</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove Sound</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="addSoundBtnTemplate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Sound</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="MondayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="TuesdayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="WednesdayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>91</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="ThursdayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Thursday</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="FridayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="SaturdayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Saturday</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="SundayCheck">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Sunday</string>
        </property>
       </widget>
       <widget class="QPushButton" name="setTemplateBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>400</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Set Template</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="templateDropdown">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="templateNameInput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>300</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QCheckBox" name="NextDayCheck">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Play Next Day</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="volumeInput">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteTemplateBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Template</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="newTemplateBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>New Template</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="updateTemplateBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>430</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Update Template</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="deconflictLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>370</y>
          <width>251</width>
          <height>81</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="SoundFiles"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>629</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCWOC_Sound_Scheduler">
    <property name="title">
     <string>CWOC Sound Scheduler</string>
    </property>
    <addaction name="actionHome"/>
    <addaction name="actionCalendar_View"/>
    <addaction name="actionTemplates"/>
   </widget>
   <addaction name="menuCWOC_Sound_Scheduler"/>
  </widget>
  <action name="actionCalendar_View">
   <property name="text">
    <string>Calendar View</string>
   </property>
  </action>
  <action name="actionTemplates">
   <property name="text">
    <string>Templates</string>
   </property>
  </action>
  <action name="actionSound_Files">
   <property name="text">
    <string>Sound Files</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
